pkg/platform/openstack/kuryr_bootstrap.go

```diff
@@ -481,6 +483,11 @@ func BootstrapKuryr(conf *operv1.NetworkSpec, kubeClient client.Client) (*bootst
 		}
 	}
 
+	err = purgeOpenStackLbPoolMember(client, poolId, addresses)
+	if err != nil {
+		return nil, errors.Wrap(err, "Failed on purging LB pool from invalid members.")
```

Comment:
Perhaps this could be rephrased to "Failed on purging invalid LB members from LB pool"
The `DeleteMember` function removes a members from a pool. [1]

[1] https://github.com/gophercloud/gophercloud/blob/master/openstack/loadbalancer/v2/pools/requests.go#L428-L429

Reply:
Ok!